go/cmd/vttablet/plugin_sysloglogger.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2016, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
You probably want to drop the copyright messages in the files your submit. Just to be sure. ;)